### ISSUE TITLE
SPRINT 5 :: Lighthouse Audit

### ISSUE THREAD

**Issue by Kristiannmiller** [ericberglund117/Refactor-tractor-WC-EB-KM-CD#56]
You must score as close to 100% as possible with the “Lighthouse Accessibility Audit” (you’ll need to deploy to GH pages for this). Be prepared to explain any accessibility audits your application is failing.
